doc/src/vtr/server_mode/index.rst

```diff
@@ -0,0 +1,203 @@
+.. _server_mode:
+
+Server Mode
+================
+
+VTR provides the ability to run in server mode using the following command-line arguments.
```

Comment:
I'd give some high-level context on what this is used for (an external client such as a UI requesting information or visualization from VPR on the compilation results of a design).

Reply:
solved https://github.com/w0lek/vtr-verilog-to-routing/pull/11/commits/97ad7f7bc84fe258c5ee3b72be95443db62dcc9d